src/main/java/org/spdx/tools/CompareSpdxDocs.java

```diff
@@ -1,20 +1,9 @@
 /**
- * Copyright (c) 2013 Source Auditor Inc.
- * Copyright (c) 2013 Black Duck Software Inc.
- *
- *   Licensed under the Apache License, Version 2.0 (the "License");
- *   you may not use this file except in compliance with the License.
- *   You may obtain a copy of the License at
- *
- *       http://www.apache.org/licenses/LICENSE-2.0
- *
- *   Unless required by applicable law or agreed to in writing, software
- *   distributed under the License is distributed on an "AS IS" BASIS,
- *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *   See the License for the specific language governing permissions and
- *   limitations under the License.
- *
-*/
+ * SPDX-FileCopyrightText: Copyright (c) 2013 Source Auditor Inc.
+ * SPDX-FileCopyrightText: Copyright (c) 2013 Black Duck Software Inc.
+ * SPDX-FileType: SOURCE
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Since this is existing code, I would prefer not to remove the Apache license headers since it is recommended practice by the Apache foundation.  For new code recently published, I'm OK not including the headers.

Reply:
Thanks Gary. I will revert those changes.

Reply:
Copyright text added back